great_expectations/data_context/types/base.py

```diff
@@ -1004,6 +1004,9 @@ class Meta:
     boto3_options = fields.Dict(
         keys=fields.Str(), values=fields.Str(), required=False, allow_none=True
     )
+    connect_args = fields.Dict(
```

Comment:
Would we be able to add `connect_args` to `ExecutionEngineConfig` class above as well?

Reply:
Otherwise the PR looks great. thanks for catching this @itaise 🙇🏼

Reply:
@Shinnnyshinshin  thank you! done :)